doc/conf.py

```diff
@@ -411,7 +411,8 @@ def setup(app):
         subprocess.run(["doxygen"], cwd=root_dir, check=True)
         doxygen_html_src = str(root_dir / "doxygen/html")
         doxygen_html_dest = str(out_dir / "doxygen")
-        subprocess.run(["cp", "-r", doxygen_html_src, doxygen_html_dest], check=True)
+        # check=False => it's fine if the source dir doesn't exist
+        subprocess.run(["cp", "-r", doxygen_html_src, doxygen_html_dest], check=False)
```

Comment:
That doesn't seem ideal because we'll also ignore errors like running out of disk or the dest doesn't exist? Checking os.exists() && os.isdir() would be a little better imo.

Reply:
Sure. I've added a check that the path exists and is a dir, and restored `check=True`.